scss/bootstrap/_variables.scss

```
@@ -880,7 +880,7 @@ $form-feedback-tooltip-border-radius: $tooltip-border-radius !default;
 
 $popover-font-size:                 $font-size-sm !default;
 $popover-bg:                        $white !default;
-$popover-max-width:                 276px !default;
+$popover-max-width:                 350px !default;
```

La valeur n'est plus par défault du coup
```suggestion
$popover-max-width:                 350px;
```